packages/grafana-ui/package.json

```diff
@@ -110,6 +110,8 @@
     "@storybook/manager-webpack5": "6.3.7",
     "@storybook/react": "6.3.7",
     "@storybook/theming": "6.3.7",
+    "@swc/core": "1.2.103",
```

Comment:
This is weird that we need to specify this here but looks like it doesn't compile without it.